src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
The problem here is this kind of use cases:

```css
@custom-selector :--blah .blah p

:--blah {
  &--wat {
```

`.blah p--active` make no sense.

So I guess it can't be handled this way...